sycl/source/detail/graph_impl.hpp

```diff
@@ -153,7 +93,10 @@ class node_impl {
   /// @param Arg Argument to lookup.
   /// @return True if \p Arg is used in node, false otherwise.
   bool has_arg(const sycl::detail::ArgDesc &Arg) {
-    for (auto &NodeArg : MArgs) {
+    // TODO: Handle types other than exec kernel
```

Comment:
Should we assert here on the type for now

Reply:
Done!